src/Listeners/CreateCouponOnStripe.php

```diff
@@ -14,6 +14,13 @@ class CreateCouponOnStripe
 
     public function handle(AffiliateCreated $event)
     {
+        /**
+         * Strip API requires minimum 
+         * 0.01 coupon
+         */
+        if( empty($this->coupon) )
```

Comment:
I believe this breaks the package if someone does want to add a coupon on Stripe for an affiliate. You'll need to change this to check if the discount is zero and then don't create a coupon.